{"uuid": "31ed3cd0-841a-471a-95ee-76a62876dedf", "children": ["185f73ff-dcfd-4035-ad63-b702e87c0949"], "befores": [{"name": "capture_screenshot_on_failure", "status": "passed", "start": 1741862114801, "stop": 1741862114801}], "afters": [{"name": "capture_screenshot_on_failure::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\Users\\rutwik\\miniconda3\\envs\\automation\\lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"C:\\Users\\rutwik\\miniconda3\\envs\\automation\\lib\\site-packages\\_pytest\\fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\rutwik\\Automation\\conftest.py\", line 121, in capture_screenshot_on_failure\n    if request.node.rep_call.failed:\n"}, "start": 1741862127790, "stop": 1741862127827}], "start": 1741862114801, "stop": 1741862127829}